test/test.py

```diff
@@ -0,0 +1,98 @@
+import munge
+import unittest
+
+
+def file_factory(n):
+    # makes a file with "the great big whale splashed the timid blue fox.\n" repeated n times
+    with open("test_files/simple_whale_" + str(n) + ".txt", "w") as out:
+        for i in range(n):
+            out.write("the great big whale splashed the timid blue fox.\n")
```

Comment:
I would usually consider this a Python constant. Good form is to declare constants at the top of the file between your imports and your first functions/classes, e.g.

```
WHALE_STRING = "the great big whale splashed the timid blue fox."
'''